src/geo/map/torque-layer.js

```diff
@@ -18,12 +18,36 @@ var TorqueLayer = LayerModelBase.extend({
     time: undefined // should be a Date instance
   },
 
-  initialize: function(attrs, options) {
+  ATTRIBUTES_THAT_TRIGGER_MAP_RELOAD: ['visible', 'sql', 'source', 'cartocss'],
+
+  initialize: function (attrs, options) {
     LayerModelBase.prototype.initialize.apply(this, arguments);
     options = options || {};
 
     this._map = options.map;
-    this.bind('change:visible change:sql change:cartocss change:source', this._reloadMap, this);
+    this.bind('change', this._onAttributeChanged, this);
+  },
+
+  _onAttributeChanged: function () {
+    var reloadMap = _.any(this.ATTRIBUTES_THAT_TRIGGER_MAP_RELOAD, function (attr) {
+      if (this.hasChanged(attr)) {
+        if (attr === 'cartocss') {
+          return this.previous('cartocss') && this._torqueTimeAttributeCartoCSSPropChanged();
+        }
+        return true;
+      }
+    }, this);
+
+    if (reloadMap) {
+      this._reloadMap();
+    }
+  },
+
+  _torqueTimeAttributeCartoCSSPropChanged: function () {
```

Comment:
Changed this in [09e10e742b714f7dc4fc7436522bb159f3c3961d](https://github.com/CartoDB/cartodb.js/pull/1268/commits/09e10e742b714f7dc4fc7436522bb159f3c3961d).